packages/http/src/validator/index.ts

```diff
@@ -75,17 +76,22 @@ const validateInputBody = (
     checkRequiredBodyIsProvided(requestBody, body),
     E.map(b => [...b, caseless(headers || {})] as const),
     E.chain(([requestBody, body, headers]) => {
+      // parse boundary string from content-type in case media type is multipart/form-data
+      const multipart = require('parse-multipart-data');
```

Comment:
We'd prefer to use `import` statements rather than dynamic `require` whenever possible.